guava/src/com/google/common/io/FileBackedOutputStream.java

```diff
@@ -54,7 +57,8 @@
       return buf;
     }
 
-    int getCount() {
+    @SuppressWarnings("return.type.incompatible") // this.getBuffer() is the same as this.buf
```

Comment:
`count` is defined in the superclass: [`ByteArrayOutputStream.java`](https://github.com/Maxi17/checker-framework/blob/master/checker/jdk/index/src/java/io/ByteArrayOutputStream.java#L56)

The warning was issued because, for example, `this.field` is treated differently from `getField()`, where `getField()` returns a reference to `field` object.
The annotation was needed because otherwise there would be errors issued in methods `openInputStream()` and `update(int)` from the same class. Both these methods call `getCount()`.